app/assets/javascripts/administrate/custom.js

```diff
@@ -1,12 +1,12 @@
 $(document).ready(function(){
-	$('#question_answer_type').on('change', function() {
-	  		// $('#form_choice').hide()
-	});
-	// $('#btn_submit_question').click(function() {
-	// 	$('#question_answer_type').prop('disabled', true);
-	// 	$('#btn_submit_question').prop('disabled', true);
-
-	// })
-
-
+	$("#serial_number").hide()
+	$("#material_material_type").change(function(){
+    var item=$(this);
+    if (item.val() == 'License') {
+  		$("#serial_number").show()
+  	}
+  	else{
+			$("#serial_number").hide()
+  	}
+  });
```

Comment:
The whitespaces here on github are uneven because it seems you mixed using tabs and spaces in your code. see here:

![Screen Shot 2019-11-18 at 15 02 49](https://user-images.githubusercontent.com/11715152/69031217-867a3100-0a14-11ea-9773-ff14879d5c21.png)

red marks are spaces, while green marks are tabs. our team's guideline is to always use spaces for indentation. 

please fix this in your code. also, you can configure your editor/IDE so that all indentation will be spaces.